src/main/java/use_cases/LoginManager.java

```diff
@@ -22,38 +26,33 @@ public LoginManager (UserManager userManager){
      * @param password The entered password
      * @return Return true if login was successful. Return false otherwise.
      */
-    public boolean login(String username, String password){
+    public boolean login(String username, String password) {
         boolean isValidLogin = verifyUser(username, password);
-//        if (isValidLogin){
-            // TODO use UserManager method to create user with all needed data
-            //currUser = [insert UserManager method] ;
-//        }
-          this.currUser = new User(username, password);
-          return true; // dummy value, remove when we implement this method
-//        return isValidLogin;
+        if (isValidLogin) {
+            //use UserManager method to create user with all needed data
+            this.currUser = this.userManager.createUser(username, password);
+        }
+        return isValidLogin;
     }
 
     /**
      * Logout current user by resetting currUser.
      */
-    public void logout(){
+    public void logout() {
         currUser = null;
```

Comment:
I think it might be good to use this.currUser here for style consistency

Reply:
Yes, you're right. Made the change